### PR TITLE
Add a new prop to ignore the collapse button on the column name bar when there's custom group header

### PR DESCRIPTION
**Pull request by CmonsterC**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Add a new prop to ignore the collapse button on the column name bar when there's custom group header

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by aaronyzhou** [comment]
@cschleiden @dzearing Can you please review this PR? We want to use this change in VSTS this sprint.

**Comment by cschleiden** [comment]
Can't we do what we do with `SelectAllVisibility`? This `ignore..` property feels one-off.

**Comment by aaronyzhou** [comment]
`SelectAllVisibility` does not control the expand/collapse all button.

**Comment by cschleiden** [comment]
@aaronyzhou I mean a similar approach.. not one property to hide something, but an enum that defaults to visible to determine whether it should be shown or not

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx [hunk 1]
```diff
@@ -366,6 +367,7 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
       getRowAriaLabel,
       checkButtonAriaLabel
     } = this.props;
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx]: add groupProps here, then you don't have to have `this.props`

File: packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts [hunk 2]
```diff
@@ -17,6 +17,11 @@ import {
 } from '../../utilities/selection/index';
 import { IViewport } from '../../utilities/decorators/withViewport';
 
+export enum CollapseAllVisibility {
+  none = 0,
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts]: should we call this `hidden`?

File: packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts [hunk 3]
```diff
@@ -177,6 +182,11 @@ export interface IGroupRenderProps {
    * Override which allows the caller to provider a customer footer.
    */
   onRenderFooter?: IRenderFunction<IGroupDividerProps>;
+
+  /**
+   * Flag to indicate whether to ignore the collapsing icon on header.
+   */
+  collapseAllVisibility?: CollapseAllVisibility;
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts]: The current default behavior is visible? If so, then you should add default props to not break existing users

Comment (aaronyzhou) [inline packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.Props.ts]: If you don't set the prop it is already default visible, but I've made it explicit.

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx [hunk 4]
```diff
@@ -42,6 +43,7 @@ export interface IDetailsHeaderProps extends React.Props<DetailsHeader> {
   onColumnContextMenu?: (column: IColumn, ev: React.MouseEvent<HTMLElement>) => void;
   onRenderColumnHeaderTooltip?: IRenderFunction<ITooltipHostProps>;
   groupNestingDepth?: number;
+  collapseAllVisibility?: CollapseAllVisibility;
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx]: line 79 in this file, don't we have to add the default value there?

**Pull request merged**